apps/teams-test-app/src/components/CalendarAPIs.tsx

```diff
@@ -1,64 +1,49 @@
 import { calendar } from '@microsoft/teams-js';
 import React, { ReactElement } from 'react';
 
-import { noHostSdkMsg } from '../App';
-import BoxAndButton from './BoxAndButton';
+import { ApiWithoutInput, ApiWithTextInput } from './utils';
 
-const CalendarAPIs = (): ReactElement => {
-  const [capabilityCheckRes, setCapabilityCheckRes] = React.useState('');
-  const [composeMeetingRes, setComposeMeetingRes] = React.useState('');
-  const [openCalendarItemRes, setOpenCalendarItemRes] = React.useState('');
+const CheckCalendarCapability = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'checkCapabilityCalendar',
```

Comment:
>checkCapabilityCalendar [](http://example.com/codeflow?start=11&length=23)

Can this be called 'checkCalendarCapability'? Since that would be consistent with its declared name.

Reply:
That would mean I need to change the E2E scenario test :( Certainly possible, but not sure it's worth it :P

Reply:
I guess I have to change a bunch of other tests anyway, so one more isn't that big of a deal. Let me make that change